### DISCUSSION TITLE
Congratulations on the release of .net 8

### DISCUSSION THREAD

**Discussion by Varorbc** [dotnet/runtime#94703] [General]
Congratulations on the release of .net 8

**Comment by ghost** [comment]
Tagging subscribers to this area: @dotnet/area-meta
See info in [area-owners.md](https://github.com/dotnet/runtime/blob/main/docs/area-owners.md) if you want to be subscribed.
<details>
<summary>Issue Details</summary>
<hr />

Today is an important milestone in my life, as I celebrate my 30th birthday. On this special day, I feel very lucky and happy, because I have the privilege of witnessing the official release of .net8. .net is an open-source cross-platform software development framework that I have been following and loving for a long time. Its history can be traced back to 2014, when it was still called dnx. Later, it was renamed as .net core, and released its first stable version in 2016. Since then, it has been constantly updated and improved, until 2020, when it was renamed again as .net. In these many years, I have been paying attention to its changes and progress, and also using it to develop my projects. I feel proud and honored for it, and also grateful for myself to be able to grow with it. Today, I am here, together with millions of .net developers around the world, to welcome the birth of .net8, and look forward to its more surprises and innovations.

<table>
  <tr>
    <th align="left">Author:</th>
    <td>Varorbc</td>
  </tr>
  <tr>
    <th align="left">Assignees:</th>
    <td>-</td>
  </tr>
  <tr>
    <th align="left">Labels:</th>
    <td>

`area-Meta`

</td>
  </tr>
  <tr>
    <th align="left">Milestone:</th>
    <td>-</td>
  </tr>
</table>
</details>
